resources/sass/app.scss

```diff
@@ -239,86 +237,86 @@ dl.import-meta .download-csv {
     display: inline-block;
-    color: $wikit-Button-normal-neutral-color;
-    border-width: $wikit-Button-border-width;
-    border-style: $wikit-Button-border-style;
-    border-color: $wikit-Button-normal-neutral-border-color;
-    border-radius: $wikit-Button-border-radius;
-    background-color: $wikit-Button-normal-neutral-background-color;
-    transition-duration: $wikit-Button-transition-duration;
-    transition-property: $wikit-Button-transition-property;
+    color: $color-base;
+    border-width: $border-width-base;
+    border-style: $border-style-base;
+    border-color: $border-color-base;
+    border-radius: $border-radius-base;
+    background-color: $background-color-interactive-subtle;
+    transition-duration: $transition-duration-base;
+    transition-property: $transition-property-base;
 
     &:focus {
-        color: $wikit-Button-normal-neutral-focus-color;
-        background-color: $wikit-Button-normal-neutral-focus-background-color;
-        border-color: $wikit-Button-normal-neutral-focus-border-color;
-        box-shadow: $wikit-Button-normal-neutral-focus-box-shadow;
+        color: $color-base;
+        background-color: $background-color-interactive-subtle;
+        border-color: $border-color-progressive--focus;
+        box-shadow: $box-shadow-inset-small $box-shadow-color-progressive--focus;
     }
 
     &:hover {
         text-decoration: none;
-        color: $wikit-Button-normal-neutral-hover-color;
-        border-color: $wikit-Button-normal-neutral-hover-border-color;
-        background-color: $wikit-Button-normal-neutral-hover-background-color;
+        color: $color-base--hover;
+        border-color: $border-color-base;
+        background-color: $background-color-base;
     }
 
     &:active {
-        color: $wikit-Button-normal-neutral-active-color;
-        border-color: $wikit-Button-normal-neutral-active-border-color;
-        background-color: $wikit-Button-normal-neutral-active-background-color;
+        color: $color-emphasized;
+        border-color: $border-color-interactive;
+        background-color: $background-color-interactive;
         box-shadow: none;
     }
 
     &.progressive {
-        color: $wikit-Button-primary-color;
-        background-color: $wikit-Button-primary-progressive-background-color;
-        border-color: $wikit-Button-primary-progressive-border-color;
+        color: $color-inverted;
+        background-color: $background-color-progressive;
+        border-color: $border-color-progressive;
 
         &:visited {
             color: white;
         }
 
         &:focus {
-            box-shadow: $wikit-Button-primary-progressive-focus-box-shadow;
-            border-color: $wikit-Button-primary-progressive-focus-border-color;
-            background-color: $wikit-Button-primary-progressive-focus-background-color;
+            box-shadow: $box-shadow-inset-small $box-shadow-color-progressive--focus $box-shadow-inset-medium $box-shadow-color-inverted;
```

Comment:
```suggestion
            box-shadow: $box-shadow-inset-small $box-shadow-color-progressive--focus, $box-shadow-inset-medium $box-shadow-color-inverted;
```

Reply:
This hopefully solves the issue with the focus inset borders. (the value was malformed)

Reply:
good catch, thanks!